display.py

```diff
@@ -15,6 +15,7 @@ def __init__(self,name,w,h):
 		pygame.display.set_icon(pygame.image.load("images/icon.png"))
 		self.display = pygame.display.set_mode((w,h))
 		self.fps     = pygame.time.Clock()
+		self.background = pygame.image.load("images/testbg.png")
```

Comment:
You haven't included the image in your PR.

Reply:
Sorry, I must have forgotten to tick it, It should be in the next commit.